### PR TITLE
Fix deployments for alternate namespaces

### PR DESCRIPTION
**Pull request by dhaiducek**
The controller and service references wouldn't work otherwise.

### CODE REVIEW

**Review by JustinKuli — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -174,7 +174,7 @@ postgres: cert-manager
 
 webhook: cert-manager
 	-kubectl create ns $(KIND_NAMESPACE)
-	sed 's/namespace: open-cluster-management/namespace: $(KIND_NAMESPACE)/g' deploy/webhook.yaml | kubectl apply -f-
+	sed -E 's,open-cluster-management(.svc|/|$$),$(KIND_NAMESPACE)\1,g' deploy/webhook.yaml | oc apply -f -
```
Comment (JustinKuli) [inline Makefile:177]: Is there a reason this uses `oc` instead of `kubectl`?

Comment (dhaiducek) [reply Makefile]: Habit 🤦‍♂️ 

It's fixed. 🙂

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**